src/Stratis.Features.SQLiteWalletRepository/TransactionsToLists.cs

```diff
@@ -36,13 +34,16 @@ public override void RecordSpend(HashHeightPair block, TxIn txIn, string pubKeyS
                 SpendTxId = spendTxId.ToString(),
                 SpendIndex = spendIndex,
                 SpendTxTotalOut = totalOut.Satoshi
-            });
+            };
+
+            this.conn.Repository.logger.LogTrace("Recording spend: {0}", prevOut.ToString());
```

Comment:
All logs like these should be `debug`.

```suggestion
            this.conn.Repository.logger.LogDebug("Recording spend: {0}", prevOut.ToString());
```